ophidian/util/Units.h

```diff
@@ -0,0 +1,107 @@
+#ifndef OPHIDIAN_UTIL_UNITS_H
+#define OPHIDIAN_UTIL_UNITS_H
+
+#include <ratio>
+#include <3rdparty/units/include/units.h>
+#include <boost/geometry/geometries/point_xy.hpp>
+
+namespace ophidian {
+namespace util {
+
+//Length Units
```

Comment:
I reviewed the code and decided to leave it that way because the namespace ophidian::util groups not only length units, but other types too. I think this way it is easier to use.